crates/gitbutler-repo/src/rebase.rs

```diff
@@ -284,32 +363,18 @@ pub fn gitbutler_merge_commits<'repository>(
 
         // in case someone checks this out with vanilla Git, we should warn why it looks like this
         let readme_content =
-        b"You have checked out a GitButler Conflicted commit. You probably didn't mean to do this.";
+            b"You have checked out a GitButler Conflicted commit. You probably didn't mean to do this.";
```

Comment:
There have been a few small formatting changes like this. How has clippy/rustfmt not complained about them?

Reply:
I know that sometimes when lines get too long, `cargo fmt` flakes out entirely and simply stops formatting that file entirely. It can be that at some point I reformatted with a longer max line length, which kind of 'jiggles' files back into place even when the `cargo fmt` run is redone with the normal file length.

Also, it's my best guess for what happened, my memory about this is vague at best.